sdcm/tester.py

```diff
@@ -420,6 +420,19 @@ def argus_collect_packages(self):
             self.log.info("Saving collected packages...")
             if len(versions) == 0:
                 self.argus_test_run.run_info.details.packages.append(self.generate_scylla_server_package())
+            if self.k8s_cluster:
```

Comment:
You forgot one more required field - the repo of the helm chart.

Reply:
What do you mean ? do we have cases that we created a k8s cluster, but there's no helm ?

Or you mean we need that report as a package ?

Do we have more then one helm repo ?

Reply:
We have `3` `helm repos` where we get `scylla operator`, `scylla manager` and `scylla` `helm charts`:
- official stable/releases -> `https://storage.googleapis.com/scylla-operator-charts/stable`
- official latest non-releases -> `https://storage.googleapis.com/scylla-operator-charts/latest`
- dev -> `https://scylla-operator-charts-dev.storage.googleapis.com/latest`
And it is possible to use any other custom one, SCT won't block it.

That's why we have dedicated SCT config option for it: `SCT_K8S_SCYLLA_OPERATOR_HELM_REPO` / `k8s_scylla_operator_helm_repo`.

For debugging needs we must have it. It directly related to the understanding of the stuff that was used in a K8S test run.

Reply:
so question is do have: 
* operator-image - docker image (scylladb/scylla-operator:tag)
* operator-helm-chart - k8s_scylla_operator_helm_repo + chart version

or:

* operator-image - docker image (scylladb/scylla-operator:tag)
* operator-helm-chart - chart version
* operator-helm-repo - k8s_scylla_operator_helm_repo

Reply:
Last one sounds to me much better.

Reply:
so we'll settle with those